apps/sms/views/conversation/js/conversation.js

```diff
@@ -1876,9 +1917,13 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    // If we are on a thread, we can call to SelectMessages and add recipients
     if (Navigation.isCurrentPanel('thread')) {
       params.items.push({
+        l10nId: 'addRecipients-label',
+        method: this.addRecipients.bind(this)
```

Comment:
I think you can have a look how you can leverage the new method `initiateNewMessage`, maybe adding a new parameter, or completing the "parameters.number" case. Or a new similar method (because the "focusComposer" boolean value would be different than what we have in `initiateNewMessage`), maybe `SwitchToNewMessage` (we call "New Message panel" this panel now, to distinguish it with the "composer" element -- we still use the 'composer' name when calling `toPanel ` though so we didn't change all places yet :) ).